### ISSUE TITLE
Using ManifestPlugin seems to not work 

### ISSUE THREAD

**Issue by amir20** [SebastianS90/webpack-polyfill-injector#9]
Hi @SebastianS90,

Thanks for the excellent library. I am trying to use this injector with `ManifestPlugin`.  My entry point looks like: 

```js
entry: {
    "details-page": `webpack-polyfill-injector?${JSON.stringify({
      modules: ["./js/details-page.js"]
    })}!`,
  },
```

Doing `ls -lah` after `webpack` shows: 

```
-rw-r--r--  1 amirraminfar  staff   152K Jan  4 07:57 details-page.js
-rw-r--r--  1 amirraminfar  staff    29K Jan  4 07:57 index.js
-rw-r--r--  1 amirraminfar  staff   7.9K Jan  4 07:57 manifest.js
-rw-r--r--  1 amirraminfar  staff    12K Jan  4 07:57 polyfills.001.js
-rw-r--r--  1 amirraminfar  staff   1.0K Jan  4 07:57 polyfills.010.js
-rw-r--r--  1 amirraminfar  staff    13K Jan  4 07:57 polyfills.011.js
-rw-r--r--  1 amirraminfar  staff   4.8K Jan  4 07:57 polyfills.100.js
-rw-r--r--  1 amirraminfar  staff    17K Jan  4 07:57 polyfills.101.js
-rw-r--r--  1 amirraminfar  staff   5.7K Jan  4 07:57 polyfills.110.js
-rw-r--r--  1 amirraminfar  staff    18K Jan  4 07:57 polyfills.111.js
-rw-r--r--  1 amirraminfar  staff   129B Jan  4 07:57 styles.js
-rw-r--r--  1 amirraminfar  staff   561K Jan  4 07:57 vendor.js
```
So it clearly works. 

However, the `manifest.json` file is:

```json
{
  "details-page.js": "js/polyfills.111.js",
  "index.js": "js/index.js",
  "manifest.js": "js/manifest.js",
  "styles.css": "css/styles.4a3f11a3c4954ae47993.css",
  "styles.js": "js/styles.js",
  "vendor.js": "js/vendor.js"
}
```

I would expect `details-page.js` to be `js/details-page.js`. 

The plugins look like:

```js
  plugins: [
    new ManifestPlugin(),
    new webpack.optimize.CommonsChunkPlugin({
      name: "vendor",
      chunks: ["details-page", "index"]
    }),
    new webpack.optimize.CommonsChunkPlugin({
      name: "manifest"
    }),
    new PolyfillInjectorPlugin({
      polyfills: ["Promise", "Array.prototype.fill", "fetch"]
    }),
    new ExtractTextPlugin("css/[name].[chunkhash].css"),
    new CleanWebpackPlugin([
      __dirname + "/clashstats/static/css",
      __dirname + "/clashstats/static/js"
    ])
  ]
```

Any idea on how to fix this?

**Comment by SebastianS90** [comment]
Thanks for this bug report. Please test so following:

Edit the file `node_modules/webpack-polyfill-injector/src/plugin.js` and comment out these two blocks:

[line 89-91](https://github.com/SebastianS90/webpack-polyfill-injector/blob/v1.0.0/src/plugin.js#L89-L91)
```js
chunksWithSingleInjector[polyfillsString].forEach((chunk) => {
    chunk.files.push(filename);
});
```

[line 104-106](https://github.com/SebastianS90/webpack-polyfill-injector/blob/v1.0.0/src/plugin.js#L104-L106)
```js
chunksWithMultiInjector[polyfillsString].forEach((chunk) => {
    chunk.files.push(outputFile);
});
```

My guess:
* You will no longer have the problem with `ManifestPlugin` :+1:
* But if you use `uglifyjs-webpack-plugin` to minify your scripts it will no longer minify the polyfills :-1:

Please confirm, then I'll look for another solution for letting uglify find those files (probably create a dedicated chunk instead of attaching the polyfills to the entry chunk).

**Comment by amir20** [comment]
I can try that this afternoon. Do you know if I change it in `node-modules`, will it pick up the changes? Or should clone the project and do `npm link`?

**Comment by SebastianS90** [comment]
Not sure about `npm link`, but when using `yarn link` together with webpack builds I often get some module not found errors. Pathing directly in `node_modules` works, but it does not persist the next install/upgrade.

(This plugin has no compiled files, I require a current `node` version that natively supports ES2015 syntax. Other plugins often transpile the sources, there you would need to apply the patches to the less-readable transpiled version of the code instead of the original source. But here you can directly patch in `src` without any recompilation.)

**Comment by amir20** [comment]
Hi, @SebastianS90. Confirmed that does fix the problem. 

```js
{
  "details-page.js": "js/details-page.js",
  "index.js": "js/index.js",
  "manifest.js": "js/manifest.js",
  "styles.css": "css/styles.4a3f11a3c4954ae47993.css",
  "styles.js": "js/styles.js",
  "vendor.js": "js/vendor.js"
}
```

I have pushed my code to a branch and you can find it at https://github.com/amir20/clashstats/tree/polyfill. `npm run build` will do production build while `webpack` will build for dev.

**Comment by SebastianS90** [comment]
I just pushed 2e437eebb2afa2d6a7488331d1cda76da11a44ee which should fix the issue, but before doing a release I will need to do further tests (tomorrow). You could already try it out in the meantime by copying the changed file into yout node_modules folder if you have time for it.

**Comment by SebastianS90** [comment]
Fixed in [`v1.0.1`](https://github.com/SebastianS90/webpack-polyfill-injector/releases/tag/v1.0.1)

**Comment by amir20** [comment]
Thanks, @SebastianS90 . I'll give it a try today. I was running in to another bug, but I'll investigate before posting an issue.
